contracts/exchange/ExchangeAccessFacet.sol

```diff
@@ -5,16 +5,21 @@ import '../lib/LibAddressSet.sol';
 import '../lib/LibPaginate.sol';
 import '../spc/SpcTopFacet.sol';
 import '../interfaces/IHasMembers.sol';
+import '../interfaces/IHasActivations.sol';
 import './lib/LibExchangeAccess.sol';
 import './lib/AExchangeFacet.sol';
 
 
 /** @title The Exchange Smart Contract.
  *  @dev The Exchange Access facet is in charge of keeping track of exchange members.
  */
-contract ExchangeAccessFacet is AExchangeFacet, IHasMembers {
+contract ExchangeAccessFacet is AExchangeFacet, IHasMembers, IHasActivations {
```

Comment:
How about `IHasDeactivatedMembers`?

Reply:
Yep, adding Member makes sense, it sounded too broad... nice!